security/putty/Portfile

```diff
@@ -19,25 +20,21 @@ homepage            http://www.chiark.greenend.org.uk/~sgtatham/putty/
 master_sites        ftp://ftp.chiark.greenend.org.uk/users/sgtatham/putty-${version}/ \
                     http://the.earth.li/~sgtatham/putty/${version}/
 
-checksums           rmd160  9357dfca867dc91be09c735b62025f9c614ab46d \
-                    sha256  bb8aa49d6e96c5a8e18a057f3150a1695ed99a24eef699e783651d1f24e7b0be
+checksums           rmd160  d4e27e59e3a7235089ccae854db2507a650da02f \
+                    sha256  419a76f45238fd45f2c76b42438993056e74fa78374f136052aaa843085beae5 \
+                    size    2619481
 
-worksrcdir          ${worksrcdir}/unix
+depends_build-append port:pkgconfig
+configure.args-append -DPUTTY_GTK_VERSION=0
 
-depends_build       port:pkgconfig
-
-# unix configure calls the configure in top folder
-patch.post_args     -d ${worksrcpath}/../
-patchfiles          patch-configure.diff
-
-configure.args-append     --without-gtk
-configure.cppflags-delete -I${prefix}/include
-configure.cppflags-append -isystem${prefix}/include
+pre-configure {
+    configure.cflags-delete -DNDEBUG
+}
```

Comment:
I don't think we want to delete this here... this flag is added by MacPorts and/or CMake